cmd/codegen/generator/go/mount.go

```diff
@@ -0,0 +1,73 @@
+package gogenerator
+
+import (
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/tonistiigi/fsutil"
+	fstypes "github.com/tonistiigi/fsutil/types"
+)
+
+// MountedFS takes a target FS and mounts it at Name
+type MountedFS struct {
+	FS   fs.FS
+	Name string
+}
```

Comment:
This little magic struct is what lets us prefix the entire FS with `internal/` so that `querybuilder` can stay public (since it's cross-imported into `cmd` as well).